Content.Server/StationEvents/Components/ImmovableRodRuleComponent.cs

```diff
@@ -9,4 +9,12 @@ public sealed partial class ImmovableRodRuleComponent : Component
 {
     [DataField("rodPrototype", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
     public string RodPrototype = "ImmovableRodKeepTilesStill";
+
+    public readonly string[] RodRandomPrototypes = { "ImmovableRodMop", "ImmovableRodShark", "ImmovableRodClown", "ImmovableRodBanana", "ImmovableRodHammer", "ImmovableRodThrongler", "ImmovableRodGibstick" };
```

Comment:
Why not use a list of prototypes that are set in YAML instead of hardcoding it?

Reply:
I attempted this, but _random.Pick did not like it and threw an error. I can look a bit closer at this and see if there's a way around it.